client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss

```diff
@@ -137,6 +137,13 @@
 
 .launchpad__progress-bar-container {
 	margin-bottom: 32px;
+
+	.launchpad__progress-bar.progress-bar.is-compact {
+		.progress-bar__progress {
+			/* stylelint-disable-next-line scales/radii */
+			border-radius: 4.5px;
```

Comment:
I assume the 4.5px is the default that was overridden before? Id imagine a % value would make more sense at a higher level if the component can be used at various sizes, but specific pixel values seem fine in this specificity level where we have it for launchpad.

Reply:
> I assume the 4.5px is the default that was overridden before?

Yeah that's exactly it

https://github.com/Automattic/wp-calypso/blob/61ee20f252625064db417ac1da60d672922aa0bb/packages/components/src/progress-bar/style.scss#L15-L23